internal/builders/bazel/build.sh

```diff
@@ -25,20 +25,23 @@ IFS=' ' read -r -a BUILD_TARGETS <<< "${TARGETS}"
 # Build with respect to entire arrays of flags and targets
 bazel build "${BUILD_FLAGS[@]}" "${BUILD_TARGETS[@]}"
 
-# Using target string, copy artifact to binaries dir
-for CURR_TARGET in "${BUILD_TARGETS[@]}"; do
-  # Take out the first two // in CURR_TARGET
-  # "//src/internal:fib" --> "src/internal:fib"
-  CD_PATH=$(echo "$CURR_TARGET" | cut -d'/' -f3-)
-
-  # Removes field after and including the colon
-  # "src/internal:fib" --> "src/internal"
-  CD_PATH=$(echo "$CD_PATH" | cut -d':' -f1)
+# Use associative array as a set to increase efficency in avoiding double copying the target
+declare -A FILES_SET
```

Comment:
If you're using variables in this file only then I think we should use lowercase by convention. Declaration and usage of variables can then be tracked by `shellcheck`.

Uppercase variable names are assumed to be global, cross-script environment variables. e.g. the ones set as inputs to this script by the workflow (`TARGETS`, `FLAGS`)

This goes for other variables too, like `BILDE_TARGETS`, `BUILD_FLAGS`, `CURR_TARGET` etc.

Apologies for not commenting on this sooner.
```suggestion
declare -A files_set
```

Reply:
Thanks for letting me know! Just switched them to the lowercase convention :)